zksync-interface/src/main.rs

```diff
@@ -0,0 +1,39 @@
+use async_trait::async_trait;
+use zksync_da_clients::{
+    types::{DAError, DispatchResponse, InclusionData},
+    DataAvailabilityClient,
+};
+
+#[derive(Debug, Clone)]
+struct EqClient;
+
+#[async_trait]
+impl DataAvailabilityClient for EqClient {
+    async fn dispatch_blob(
+        &self,
+        batch_number: u32,
+        data: Vec<u8>,
+    ) -> Result<DispatchResponse, DAError> {
+        todo!()
+    }
+
+    async fn get_inclusion_data(&self, blob_id: &str) -> Result<Option<InclusionData>, DAError> {
+        todo!()
+    }
```

Comment:
likely most impls are already done in https://github.com/matter-labs/zksync-era/blob/f3104a8149068d6d1c34088b0345b06b7acb68dc/core/node/da_clients/src/celestia/client.rs#L56 

`get_inclusion_data` being the main thing provided as an example in this PR

Reply:
I'd rather we just import the zkstack crate, rather than copy paste

Reply:
in which case we will need to move the `sdk/` into zksync-era, as I've done in the [PR](https://github.com/matter-labs/zksync-era/pull/3553), because you can't have a circular dependency